content/22.vaccines.md

```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
 These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
 Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
 
+The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
+Participants that received the recombinant SARS-CoV-2 vaccine with or without Matrix-M adjuvant got two injections, 21 days apart.
+Primary outcomes that were assessed include reactogenicity, lab-values (serum chemistry and hematology), and anti-spike IgG levels.
+Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
+The authors reported that no serious treatment-related adverse events occurred in any of the treatment arms.
+Reactogenicity was mostly absent and of short duration.
+The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in the convalescent plasma of symptomatic patients.
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.
```

Comment:
Is the placebo just the adjuvant I assume...just glancing through the paper I think thats the case, but just wanted to confirm with you

Reply:
Hmmm I'm not sure they explain what it was! I might leave a comment to ask the authors. There was another clinical trial of the adjuvant alone but it's not clear what they did here.

For now, I'm making a note here so we can merge this in.
```suggestion
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.<!--To Do: What is placebo?-->
```

Reply:
I think adjuvant only was done previously already. The company uses this adjuvant for other vaccines as well so they did not have to test the adjuvant on its own anymore.